### PR TITLE
New module product contract variable qty

### PR DESCRIPTION
**Pull request by sbejaoui**
This module extend `product_contract` and `contract_variable_qty` modules to be able to set default value for `qty_type` `qty_formula_id` for contract products to be used in sale order and contract line creation.

### CODE REVIEW

**Review by ThomasBinsfeld — APPROVED** [review]
Conflicts

**Comment by ThomasBinsfeld** [comment]
Rebased on 12.0

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by sbejaoui** [comment]
@gva-acsone, @rafaelbn , can you review this please

**Review by gva-acsone — APPROVED** [review]
Functional test +1

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM.

At least change README as some content is irrelevant or add readme section + a better usage description

**Review by hparfr — APPROVED** [review]

File: product_contract_variable_quantity/README.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+==================================
+Product Contract Variable Quantity
+==================================
+
+Product contract with variable quantity
+
+Installation
+============
+
+To install this module, you need to:
```
Comment (pedrobaeza) [inline product_contract_variable_quantity/README.rst:14]: Complete README

**Review by pedrobaeza — DISMISSED** [review]

File: product_contract_variable_quantity/README.rst (hunk 1 above)
Comment (sbejaoui) [reply product_contract_variable_quantity/README.rst:14]: There readme will be overridden after merge.

You mean add more info to readme fragments?

Comment (pedrobaeza) [reply product_contract_variable_quantity/README.rst:14]: Ah, OK, I didn't see the fragments.

**Pull request merged**